README.md

```diff
@@ -2,25 +2,12 @@
 
```

Comment:
- [ ] I could see we added .gitignore, let's make sure we add `node_modules` and `test.md` as per project requirements. anything in this file will make git ignores it from tracking it. you could read more about it https://git-scm.com/docs/gitignore